src/main/java/io/github/moulberry/notenoughupdates/overlays/FarmingSkillOverlay.java

```diff
@@ -310,6 +402,16 @@ public void updateFrequent() {
 
 			NumberFormat format = NumberFormat.getIntegerInstance();
 
+			//Stands for 'Crops Per Hour Or Per Minute'
+			String crphopm = NotEnoughUpdates.INSTANCE.config.skillOverlays.cropsPerHour ? "h" : "m";
```

Comment:
I'm sure you can find a better name for this? For example: `cropsTimeSuffix`

Reply:
That is an excellent name, in my defense, it was 2am (will change lol)